{"name": "test2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 1\n  +2\n  -1", "trace": "self = <test_case.test_weather_condition.TestWeatherCondition object at 0x109d23518>\n\n    def test2(self):\n>       assert 2 == 1\nE       assert 2 == 1\nE         +2\nE         -1\n\ntest_case/test_weather_condition.py:50: AssertionError"}, "steps": [{"name": "添加失败截图...", "status": "passed", "attachments": [{"name": "失败截图", "source": "365e9004-fec2-4811-89d4-43e605a86c46-attachment.png", "type": "image/png"}], "start": 1585192380020, "stop": 1585192383060}], "start": 1585192377549, "stop": 1585192377550, "uuid": "32f39c08-5bfc-459e-b3ba-6658c7c23a3f", "historyId": "3f9bf41955f208347948ea1486ce3359", "fullName": "test_case.test_weather_condition.TestWeatherCondition#test2", "labels": [{"name": "parentSuite", "value": "test_case"}, {"name": "suite", "value": "test_weather_condition"}, {"name": "subSuite", "value": "TestWeatherCondition"}, {"name": "host", "value": "liqingMacBook-Pro.local"}, {"name": "thread", "value": "33455-Thread-2"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_weather_condition"}]}